suites/pacific/upgrades/tier-1_upgrade_test-4x-to-5x-container_custom_name.yaml

```diff
@@ -0,0 +1,121 @@
+    polarion-id: CEPH-10961
+    module: test_ansible.py
+    config:
+      use_cdn: True
+      build: '4.x'
+      ansi_config:
+        ntp_service_enabled: true
+        ceph_origin: repository
+        ceph_repository: rhcs
+        ceph_repository_type: cdn
+        ceph_rhcs_version: 4
+        dmcrypt: True
+        osd_scenario: lvm
+        osd_auto_discovery: False
+        containerized_deployment: true
+        ceph_docker_image: "rhceph/rhceph-4-rhel8"
+        ceph_docker_image_tag: "latest"
+        ceph_docker_registry: "registry.redhat.io"
+        copy_admin_key: true
+        dashboard_enabled: True
+        dashboard_admin_user: admin
+        dashboard_admin_password: p@ssw0rd
+        grafana_admin_user: admin
+        grafana_admin_password: p@ssw0rd
+        node_exporter_container_image: registry.redhat.io/openshift4/ose-prometheus-node-exporter:v4.6
+        grafana_container_image: registry.redhat.io/rhceph/rhceph-4-dashboard-rhel8:4
+        prometheus_container_image: registry.redhat.io/openshift4/ose-prometheus:v4.6
+        alertmanager_container_image: registry.redhat.io/openshift4/ose-prometheus-alertmanager:v4.6
+    desc: deploy ceph containerized 4.x cdn setup using ceph-ansible
+    destroy-cluster: False
+    abort-on-fail: true
+
+- test:
+    name: check-ceph-health
+    module: exec.py
+    config:
+      cmd: ceph -s
+      sudo: True
+    desc: Check for ceph health debug info
+
+- test:
+     name: Upgrade along with IOs
+     module: test_parallel.py
+     parallel:
+       - test:
+           name: rados_bench_test
+           module: radosbench.py
+           config:
+             pg_num: '128'
+             pool_type: 'normal'
+           desc: run rados bench for 360 - normal profile
+       - test:
+           name: rbd-io
+           module: rbd_faster_exports.py
+           config:
+             io-total: 100M
+             cleanup: false
+           desc: Perform export during read/write,resizing,flattening,lock operations
+       - test:
+           name: Upgrade containerized ceph to 5.x latest
+           polarion-id: CEPH-83575198
+           module: test_ansible_upgrade.py
+           config:
+             build: '5.x'
+             ansi_config:
+               ceph_origin: distro
+               ceph_repository: rhcs
+               ceph_rhcs_version: 5
+               osd_scenario: lvm
+               osd_auto_discovery: False
+               dmcrypt: True
+               fetch_directory: ~/fetch
+               copy_admin_key: true
+               containerized_deployment: true
+               upgrade_ceph_packages: True
+               dashboard_enabled: True
+               dashboard_admin_user: admin
+               dashboard_admin_password: p@ssw0rd
+               grafana_admin_user: admin
+               grafana_admin_password: p@ssw0rd
+               node_exporter_container_image: registry.redhat.io/openshift4/ose-prometheus-node-exporter:v4.6
```

Comment:
In 5.2 onwards monitoring stack images got updated to `v4.10` refer: https://bugzilla.redhat.com/show_bug.cgi?id=2065443#c6
hence can you please use the same images here?

Reply:
Ack. Noted, the same shall be updated